recipe/meta.yaml

```
@@ -1,56 +1,54 @@
 {% set name = "usaddress" %}
-{% set version = "0.5.10" %}
+{% set version = "0.5.14" %}
 
 package:
-  name: {{ name }}
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  # tests and license are not in the tarball on pypi
-  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: 1a8ebf62d0cce58d7d8286dde70373c530a9317b6fe1752a4197b75b7d0870e3
-  - url: https://github.com/datamade/usaddress/archive/refs/tags/v0.5.10.tar.gz
-    sha256: ef83b69900fe490510fff6aa7239eef1b6a4ea4d00482c73ed4e75096da20f72
-    folder: other
+  - url: https://github.com/datamade/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: a29b1bcaada285684948be1d1c19cf9ed7070284bb3ac0bb9a92531922d50982
 
 build:
-  number: 1002
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: True  # [py<39]
 
 requirements:
   host:
     - python
     - pip
-    - setuptools
+    - setuptools >=42
     - wheel
+    - parserator
+    - probableparsing
   run:
     - python
-    # requests is only used in an isolated script import_osm.py
-    # - requests
-    - future >=0.14
     - python-crfsuite >=0.7
     - probableparsing
 
 test:
   source_files:
-    - other/tests
+    - tests
+    - measure_performance/test_data
   imports:
     - usaddress
   requires:
     - pip
     - pytest
+    - parserator
   commands:
     - pip check
-    # ignore test_labeling.py because it depends on test files and it's about performance,
-    # and test_tagging.py because one must train the model (parserator train --trainfile FILES) 
-    # to create the usaddr.crfsuite file before one can use the parse and tag methods
-    - pytest other/tests --ignore=other/tests/test_labeling.py
+    - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
+    - python -c "import usaddress; addr='123 Main St. Suite 100 Chicago, IL'; usaddress.parse(addr); usaddress.tag(addr)"
+    - cp -r measure_performance/test_data "${SP_DIR}"
+    - pytest -v tests
 
 about:
-  home: https://parserator.datamade.us/usaddress/
+  home: https://parserator.datamade.us/usaddress
```

https://github.com/datamade/usaddress/blob/v0.5.14/pyproject.toml#L27

```suggestion
  home: https://github.com/datamade/usaddress
```